src/StepInput/StepInput.js

```diff
@@ -0,0 +1,140 @@
+import Button from '../Button/Button';
+import classnames from 'classnames';
+import { FORM_MESSAGE_TYPES } from '../utils/constants';
+import FormInput from '../Forms/FormInput';
+import FormMessage from '../Forms/_FormMessage';
+import InputGroup from '../InputGroup/InputGroup';
+import keycode from 'keycode';
+import PropTypes from 'prop-types';
+import React, { useCallback, useState } from 'react';
+
+const StepInput = React.forwardRef(({
+    children,
+    className,
+    disabled,
+    disableStyles,
+    placeholder,
+    readOnly,
+    validationState,
+    value,
+    ...rest
+}, ref) => {
+    const [inputValue, updateInputValue] = useState(value);
+
+    const inputGroupClasses = classnames(
+        className,
+        'fd-input-group--control',
+        {
+            'is-disabled': disabled,
+            [`is-${validationState?.state}`]: validationState ?.state
+        }
+    );
+
+    const ifValeEqualsSign = (val) => val === '' || val === '+' || val === '-';
+
+    const decreaseValue = useCallback(() => {
+        if (ifValeEqualsSign(inputValue)) {
+            return;
+        }
+        updateInputValue(inputValue - 1);
+    });
+
+    const increaseValue = useCallback(() => {
+        if (ifValeEqualsSign(inputValue)) {
+            return;
+        }
+        updateInputValue(inputValue + 1);
+    });
+
+    const onChangeInputValue = useCallback((event) => {
+        const currentValue = event.target.value;
+        const isNumber = !isNaN(parseInt(currentValue, 10));
+        if (isNumber) {
+            updateInputValue(parseInt(currentValue, 10));
+        } else if (ifValeEqualsSign(currentValue)) {
+            updateInputValue(currentValue);
+        }
+    });
+
+    const onKeyDownInput = useCallback((event) => {
+        switch (keycode(event)) {
+            case 'up':
+                increaseValue();
+                event.preventDefault();
+                event.stopPropagation();
+                break;
+            case 'down':
+                decreaseValue();
+                event.preventDefault();
+                event.stopPropagation();
+                break;
+            default:
+        }
+    });
+
+    const minusBtn = (
+        <InputGroup.Addon isButton>
+            <Button disabled={disabled}
```

Comment:
Can you add an aria-label here with some "Step down" and "Step up" defaults - using the `localizedText` prop like seen here: https://github.com/SAP/fundamental-react/blob/e3e6c9ae7986085a4ce65b39d2da853db258fd06/src/Counter/Counter.js#L36

Reply:
Added,please review.